packages/@okta/vuepress-theme-prose/index.js

```diff
@@ -14,9 +14,9 @@ module.exports = (options, ctx) => {
       $page.fullHeaders = resolveHeaders($page);
 
       // Set to true to enable the redesign in development
-      const redesign = process.env.REDESIGN || false;
+      //const redesign = process.env.REDESIGN || false;
       // Respect the redesign flag unless this is a production build
-      $page.redesign = !ctx.isProd && redesign;
+      $page.redesign = true;
```

Comment:
Note

Reply:
just for my understanding, does this mean we're ignoring the redesign flag here from now on?

Reply:
Ah no, sorry this is just turned on in the PR for the Netlify deploy. Will be turning it off before merging @kamaldeepcheema-okta

Reply:
ah ok, thanks for the clarification!